### ISSUE TITLE
can't copy/cut node when select inline-void element alone 

### ISSUE THREAD

**Issue by githoniel** [ianstormtaylor/slate#4104]
**Description**
can't copy/cut node when select inline-void element alone 

**Recording**
just make a inline void element and press ctrl+c， it will not by copyed

**Sandbox**

https://codesandbox.io/s/slate-reproductions-forked-1omtu?file=/index.js

1. select image only and ctrl+c, it will not by copyed
2. select image with text, it will be copyed

**Expectation**

after investigation， I found that inline void element will rendered with `contentEditable={false}`

```ts
//slate-react
// src/components/element.tsx
if (Editor.isVoid(editor, element)) {
    attributes['data-slate-void'] = true

    if (!readOnly && isInline) {
      attributes.contentEditable = false
    }
}

this will prevent onCopyEvent to trigger

```

**Environment**
- Slate Version: 0.60
- Operating System: win
- Browser: chrome

**Comment by nivekithan** [comment]
I dont think this is a bug. In this [link](https://codesandbox.io/s/slate-reproductions-forked-hl8fr) instead of rendering image I rendered type of element. If you try copying and pasting it you can see that it works as expected. 

It works because we are copying and pasting text content so it works but when copying and pasting `image` we have to explicitly tell how to render these type of content.

For reference you can take a look at source code of image  [examples](https://github.com/ianstormtaylor/slate/blob/main/site/examples/images.tsx)

**Comment by githoniel** [comment]
> I dont think this is a bug. In this [link](https://codesandbox.io/s/slate-reproductions-forked-hl8fr) instead of rendering image I rendered type of element. If you try copying and pasting it you can see that it works as expected.
> 
> It works because we are copying and pasting text content so it works but when copying and pasting `image` we have to explicitly tell how to render these type of content.
> 
> For reference you can take a look at source code of image [examples](https://github.com/ianstormtaylor/slate/blob/main/site/examples/images.tsx)

Image element is  *block* void element in examples while in this issue it is *inline* void element
